### PR TITLE
Update iOS release notes template.

### PR DESCRIPTION
**Pull request by tjprescott**
Follow up to https://github.com/Azure/azure-sdk/pull/2984

### CODE REVIEW

File: eng/scripts/release-template/ios.md [hunk 1]
```diff
@@ -28,22 +28,31 @@ To install the latest GA and beta libraries, we recommend you use the [Swift Pac
...
-Open your project's `Package.swift` file and add a new package dependency to your project's `dependencies` section, specifying the clone URL of the repository and the version specifier you wish to use:
+Open your project's `Package.swift` file and add a new package dependency to your project's `dependencies` section, specifying the clone URL of the Swift Package Manager mirror repository and the version specifier you wish to use:
 
 ```swift
-// Insert dependencies here
+// swift-tools-version:5.3
+dependencies: [
+    ...
+    .package(name: "AzureCore", url: "https://github.com/Azure/SwiftPM-AzureCore.git", from: "1.0.0-beta.12")
+]
```
Comment (vcolin7) [inline eng/scripts/release-template/ios.md:44]: I assume you added `AzureCore` to have an example, is that right?

**Review by vcolin7 — APPROVED** [review]

**Review by weshaggard — APPROVED** [review]

File: eng/scripts/release-template/ios.md (hunk 1 above)
Comment (tjprescott) [reply eng/scripts/release-template/ios.md:44]: Exactly. Better than an empty comment

**Comment by weshaggard** [comment]
@tjprescott @vcolin7 by the way I think we might have enough infrastructure in place now that we might be able to start automatically pulling the ios release notes. If you guys are interested in trying that out let me know and we can test it out.

**Comment by tjprescott** [comment]
@weshaggard definitely interested 😄

**Comment by weshaggard** [comment]
@tjprescott have a look at PR https://github.com/Azure/azure-sdk/pull/2995. That is the data that our automation was able to pull from the changelogs in for the ios repo. If that data looks about right then we need to update the ios.md template here to start pulling in the data from the ios.yml file that we can generate. Feel free to reach out to me and we can work through that together.

**Comment by tjprescott** [comment]
@weshaggard you mean #2997?  It looks correct to me!

**Comment by weshaggard** [comment]
Yes I did mean #2997. Please work with myself or @chidozieononiwu to get your MD template updated and we can then just start consuming the data from the ios.yml file.

**Comment by weshaggard** [comment]
I'm not sure who is working on the android release notes but we should be able to support them now as well.

**Pull request merged**